tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,9 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert result == False
```

Comment:
Something to consider, if the function returns `False` does that guarantee the function did not alter the `inventory`? Checking the `vendor.inventory` contents as well would confirm there are no unintended side effects. 
- What could that look like?